src/test/java/com/oracle/nosql/spring/data/test/TestTTL.java

```diff
@@ -47,6 +47,11 @@ public void setup() {
         template.dropTableIfExists(EntityWith10DaysTTL.class.getSimpleName());
         template.dropTableIfExists(EntityWithDefaultTTL.class.getSimpleName());
         template.dropTableIfExists(EntityWithNegativeTTL.class.getSimpleName());
+
+        template.createTableIfNotExists(template.
```

Comment:
Actually table is automatically created during @AutoWired of repository. But, I want to make sure that table does not have any data. This can happen if test fails and table is not deleted from the db and can have rows. That is why forcefully delete and re-create here